test_scripts/API/Navigation/SendLocation/011_SendLocation_whitespaces.lua

```diff
@@ -0,0 +1,116 @@
+
+-- Expected:
+
+-- SDL validates parameters of the request
+-- SDL responds INVALID_DATA, success: false
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSendLocation = require('test_scripts/API/SendLocation/commonSendLocation')
+
+--[[ Local Variables ]]
+local stringParams = {
+    address = {
+        countryName = "a",
+        countryCode = "a",
+        postalCode = "a",
+        administrativeArea = "a",
+        subAdministrativeArea = "s",
+        locality = "a",
+        subLocality = "s",
+        thoroughfare = "s",
+        subThoroughfare = "s"
+    },
+    locationName ="a",
+    locationDescription ="a",
+    addressLines = {"hello"}, 
+    phoneNumber ="123456789",
+    locationImage = 
+    { 
+        value ="icon.png",
+    }
+}
+
+--[[ Local Functions ]]
+local function sendLocation(params, self)
+    params["longitudeDegrees"] = 60
+    params["latitudeDegrees"] = 60
+    params["deliveryMode"] = "PROMPT"
+    params["locationImage"]["imageType"] = "DYNAMIC"
+    
+    local cid = self.mobileSession1:SendRPC("SendLocation", params)
+
+    EXPECT_HMICALL("Navigation.SendLocation"):Times(0)
+
+    self.mobileSession1:ExpectResponse(cid, { success = false, resultCode = "INVALID_DATA"})
+
+    commonSendLocation.delayedExp()
+end
+
+local function put_file(self)
+    local cid = self.mobileSession1:SendRPC("PutFile", {
+                    syncFileName = "icon.png", 
+                    fileType = "GRAPHIC_PNG", 
+                    persistentFile = false, 
+                    systemFile = false},
+                "files/icon.png")
+
+    self.mobileSession1:ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonSendLocation.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", commonSendLocation.start)
+runner.Step("RAI, PTU", commonSendLocation.registerApplicationWithPTU)
+runner.Step("Activate App", commonSendLocation.activateApp)
+runner.Step("Upload file", put_file)
+
+runner.Title("Test - empty strings")
+for key,value in pairs(stringParams) do
+    local parametersToSend = stringParams
+    if (type(value) == "table") then
+        for subKey,subValue in pairs(value) do
+            parametersToSend[key][subKey] = ""
+            runner.Step("SendLocation-invalid-type-of-" .. tostring(subKey), sendLocation, {parametersToSend})
+        end
+    else
+        parametersToSend[key] = ""
+        runner.Step("SendLocation-invalid-type-of-" .. tostring(key), sendLocation, {parametersToSend})
+    end
+end
+
+runner.Title("Test - whitespaces")
+for key,value in pairs(stringParams) do
+    local parametersToSend = stringParams
+    if (type(value) == "table") then
+        for subKey,subValue in pairs(value) do
```

Comment:
@Itileda Variable 'subValue' is not used

Reply:
@dboltovskyi , fixed in 6f2e7484bc3d0171c93beefc946839185e141366